tests/Feature/CountryControllerTest.php

```diff
@@ -6,15 +6,15 @@
 
 use App\Models\Country;
 use Generator;
-use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\LazilyRefreshDatabase;
 use Inertia\Testing\AssertableInertia as Assert;
 use Tests\TestCase;
 
 class CountryControllerTest extends TestCase
 {
-    use RefreshDatabase;
+    use LazilyRefreshDatabase;
```

Comment:
Remove this trait from other classes and move to `TestCase`. It will be inherited.

`use` keyword inside class is for Traits
https://www.php.net/manual/en/language.oop5.traits.php
https://www.educative.io/answers/what-is-trait-in-php
https://www.hashbangcode.com/article/traits-versus-inheritance-php

https://www.phptutorial.net/php-oop/php-inheritance/
https://www.php.net/manual/en/language.oop5.php